pom.xml

```diff
@@ -53,6 +54,12 @@
     </distributionManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>${spring-version}</version>
```

Comment:
We'd want this marked as a provided dependency so that it doesn't get bundled with the rest of the library, but still satisfies compile-time dependencies.

Reply:
Done in the new commits.